hvillacrusisCT/dotTheory---Javascript
# JS-691 Build an Object Challenge

## Related Ticket
[JS-691](https://codeandtheory.atlassian.net/browse/JS-691)

## Description
Added the Pet Directory functionality to the application, allowing users to view a list of pets with their details.

## Changes
- Created an array of `pet` objects in `pets.js`.
- Each object includes properties for `name`, `type`, `breed`, `age`, and `photo`.
- Added a `displayPets` function that loops through the `pets` array and dynamically adds pet details to the HTML.

## Validation
1. Make sure the `pets.js` file is in the same directory as the `index.html`.
2. Open the `pets.js` file.
3. The `pets` array should be present with pet objects.
4. Each pet object should include properties for `name`, `type`, `breed`, `age`, and `photo`.
5. The `displayPets` function should be present.
6. Open the `index.html` file in a web browser. This should automatically display the pet directory from the `pets.js` file.
7. The pet directory should include the name, type, breed, age, and photo for each pet.
8. The photos should display correctly and the image source should match the `photo` property for each pet object in the `pets.js` file.
